src/uptane/root.cc

```diff
@@ -48,12 +48,16 @@ Root::Root(const std::string &repository, const Json::Value &json) : policy_(kCh
     // Threshold
     int64_t requiredThreshold = (*it)["threshold"].asInt64();
     if (requiredThreshold < kMinSignatures) {
+      // (int) is to stop << taking a reference to kMinSignatures
+      // http://www.stroustrup.com/bs_faq2.html#in-class
       LOG_DEBUG << "Failing with threshold for role " << role << " too small: " << requiredThreshold << " < "
-                << kMinSignatures;
+                << (int)kMinSignatures;
```

Comment:
C style cast is not allowed now. clang-tidy will complain.

Reply:
Ah, thank you!